packages/plugins/plugin-client/src/translations.ts

```diff
@@ -9,6 +9,12 @@ export default [
     'en-US': {
       [CLIENT_PLUGIN]: {
         'open shell label': 'Open HALO',
+        'recovery code dialog title': 'Recovery Code',
+        // TODO(wittjosiah): Update this.
```

Comment:
@Raymmar @richburdon this is intended as placeholder copy, open to suggestions for best thing to say here.